src/runtime/GlobalObject.h

```diff
@@ -250,6 +250,8 @@ class FunctionObject;
     F(temporal, Object, objName)                       \
     F(temporalCalendar, FunctionObject, objName)       \
     F(temporalCalendarPrototype, Object, objName)      \
+    F(temporalDuration, FunctionObject, objName)       \
+    F(temporalDurationPrototype, Object, objName)      \
     F(temporalPlainDate, FunctionObject, objName)      \
     F(temporalPlainDatePrototype, Object, objName)     \
     F(temporalPlainTime, FunctionObject, objName)      \
```

Comment:
I'm wondering that these defined elements in `GlobalObject` are needed?

Reply:
They are used in `BuiltinTemporal.cpp`. Can I just define it in `installTemporal` function with `new NativeFunctionObject` and use it?

Reply:
Yes, please do so.
It seems that we don't need to define these function objects in GlobalObject.